test/dependency_source/test_requirement.py

```diff
@@ -169,6 +169,20 @@ def test_requirement_source_non_editable_without_egg_fragment(monkeypatch):
     )
 
 
+@pytest.mark.online
```

Comment:
Does this test need to be online? From the body it looks like it shouldn't need network access 🙂

Reply:
Oops! I'll fix this in #488.